pacman/model/partitioner_splitters/splitter_one_app_one_machine.py

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) 2020-2021 The University of Manchester
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import logging
+from spinn_utilities.overrides import overrides
+from spinn_utilities.log import FormatAdapter
+from pacman.exceptions import PacmanConfigurationException
+from pacman.model.graphs.machine import MachineEdge
+from pacman.model.partitioner_splitters.abstract_splitters import (
+    AbstractSplitterCommon)
+from pacman.model.graphs.application.abstract import (
+    AbstractOneAppOneMachineVertex)
+
+logger = FormatAdapter(logging.getLogger(__name__))
+
+
+class SplitterOneAppOneMachine(AbstractSplitterCommon):
+
+    NOT_SUITABLE_VERTEX_ERROR = (
+        "The vertex {} cannot be supported by the {} as "
+        "the vertex does implement AbstractOneAppOneMachineVertex")
```

Comment:
I think you mean "does not implement" rather than "does implement"?

Reply:
fixed